### PR TITLE
Update build-windows.md

### PR DESCRIPTION
**Pull request by WhiteSymmetry**

### CODE REVIEW

File: doc/build-windows.md [hunk 1]
```diff
@@ -108,6 +108,7 @@ Acquire the source in the usual way:
     git clone https://github.com/bitcoin/bitcoin.git
 
 Once the source code is ready the build steps are below.
+cd bitcoin
```
Comment (MeshCollider) [inline doc/build-windows.md]: The indentation needs to match the other steps below for this to be formatted correctly :)

Comment (randolf) [reply doc/build-windows.md]: @WhiteSymmetry If you need some help in understanding what needs to be done here, please feel free to ask.  (I agree with @MeshCollider's review and request.)

Comment (laanwj) [reply doc/build-windows.md]: Take a look at the rendered output: https://github.com/WhiteSymmetry/bitcoin/blob/527a0b805fb3d58ce57f86d795d2de677d1677ac/doc/build-windows.md#building-for-64-bit-windows  to see how this goes wrong.

Comment (instagibbs) [reply doc/build-windows.md]: during review you can also quick "view" button in code changes section to see the formatting

**Comment by laanwj** [comment]
Closing, this is taking too long for a one-line change like this. If you intend to work on this again let me know and I'll reopen.

**Pull request closed without merging**